packages/cloudflare/src/cli/commands/build.ts

```diff
@@ -0,0 +1,48 @@
+import { compileOpenNextConfig } from "@opennextjs/aws/build/compileConfig.js";
+import type yargs from "yargs";
+
+import { build as buildImpl } from "../build/build.js";
+import { createOpenNextConfigIfNotExistent } from "../build/utils/create-config-files.js";
+import { setupCLI, withWranglerOptions, withWranglerPassthroughArgs } from "./setup-cli.js";
+
+async function buildCommand(args: {
+	wranglerArgs: string[];
+	config: string | undefined;
```

Comment:
nit: rename this to `configPath` (we also use `config` for the values in i.e. `populateR2IncrementalCache`)